plugins/permission-react/src/components/PermissionedRoute.test.tsx

```diff
@@ -26,10 +27,10 @@ const mockUsePermission = usePermission as jest.MockedFunction<
   typeof usePermission
 >;
 
-const permission = {
+const permission = createPermission({
   name: 'access.something',
   attributes: { action: 'read' as const },
```

Comment:
i believe we can drop the `as const` modifiers on all `action` values now that we have a `createPermission` factory

Reply:
Yep, totally! Took a pass through and I think it's just this one that has it - removed in cd1d7ff233.